src/dodal/devices/electron_analyser/specs_analyser_controller.py

```diff
@@ -0,0 +1,14 @@
+from ophyd_async.epics.core import epics_signal_rw
+
+from dodal.devices.electron_analyser.abstract_analyser_controller import (
+    AbstractAnalyserController,
+)
+
+
+class SpecsAnalyserController(AbstractAnalyserController):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        super().__init__(prefix, name)
```

Comment:
Must: The `super()` call needs to be at the end otherwise the children do not get properly named and so when you read the analyser you do not get the `psu_mode` etc.

```python
async def test_device_has_expected_reading(
    sim_analyser: SpecsAnalyserController,
) -> None:
    reading = await sim_analyser.read()
    assert "analyser-psu_mode" in reading.keys()
```
Or, even more completely, using https://github.com/bluesky/ophyd-async/blob/2984ea49b307b661f8e9c32b66208c0d8da1940e/src/ophyd_async/testing/_assert.py#L45